autoarray/plot/mat_wrap/include.py

```diff
@@ -0,0 +1,168 @@
+from autoconf import conf
+from autoarray.structures import abstract_structure, grids
+import typing
+
+
+class AbstractInclude:
+    def __init__(
+        self, origin: typing.Optional[bool] = None, mask: typing.Optional[bool] = None
+    ):
+        """
+        Sets which `Visuals` are included on a figure that is plotted using a `Plotter`.
+
+        The `Include` object is used to extract the visuals of the plotted data structure (e.g. `Array`, `Grid`) s so they can be used in plot
+        functions. Only visuals with a `True` entry in the `Include` object are extracted and therefore plotted.
+
+        If an entry is not input into the class (e.g. it retains its default entry of `None`) then the bool is
+        loaded from the `config/visualize/include.ini` config file. This means the default visuals of a project
+        can be specified in a config file.
+
+        Parameters
+        ----------
+        origin : bool
+            If `True`, the `origin` of the plotted data structure (e.g. `Array`, `Grid`)  is included on the figure.
+        mask : bool
+            if `True`, the `mask` of the plotted data structure (e.g. `Array`, `Grid`)  is included on the figure.
+        """
+
+        self._origin = origin
+        self._mask = mask
+
+    def load(self, value, name):
+        if value is True:
+            return True
+        elif value is False:
+            return False
+        elif value is None:
+            return conf.instance["visualize"]["include"]["include"][name]

```

Comment:
What if value is neither True, False or None? Should you return the value? Should it raise an exception?

Reply:
I'll get it to raise an exception.